app/views/parks/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if params[:search].present? %>
```

Comment:
There was also the thought of on sign up the user would provide the city they live in and then when they visit the parks path it will automatically show parks in the city they live in.